cli/pom.xml

```diff
@@ -92,6 +92,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>findbugs</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <phase>verify</phase>
+            <configuration>
+              <effort>Max</effort>
+              <threshold>High</threshold>
+              <!--Excludes file is located on the top level-->
+              <excludeFilterFile>../src/findbugs/findbugs-excludes.xml</excludeFilterFile>
```

Comment:
so - did you check this works when you `cd core && mvn verify` and `mvn pl core verify` because I believe one of these will fail due to looking for a relative path from the [PC]WD and not pick up the exclusion correctly.